recipes.json

```diff
@@ -0,0 +1,332 @@
+      "18. When both the sides have golden brown spots and are cooked, remove it from heat. As one parotta is cooked, shape the next one.",
+      "19. Once all the parottas are cooked, stack them on the kitchen counter and press them from both sides to bring out the layers. This has to be done when the parottas are hot/warm enough.",
+      "20. Serve hot with Salna or Kurma with a side of thayir pachadi/onion raita. If not serving immediately, store them in hotpack/hotbox to retain the softness. You can also freeze the cooked parottas in ziplock bags for upto few weeks."
+    ],
+    "reviews": {
+      "average_rating": 4.0,
+      "user_reviews": [
+        {
+          "username": "Tharun",
+          "rating": 4,
+          "comment": "Very refreshing taste!! Recommend for all."
+        },
+        {
+          "username": "Ajith",
+          "rating": 4,
+          "comment": "Great flavor combination. Took me to my childhood!"
+        }
+      ]
+    },
+    "tags": ["2 hours", "1 serving", "Intermediate"]
+  },
+  {
+    "id": "2",
+    "name": "Indonesian Chicken Noodles",
+    "image": "images/recipiepage/noodles.jpg",
+    "description": "This soup is brimming with a rich symphony of flavours from aromatic ingredients like ginger, garlic, shallots, and turmeric, lending it a unique colour and flavour profile. Mixed with noodles, potatoes, and a variety of veggies, this combination results in a delicious, filling soup that everyone can enjoy.",
+    "time": "40 minutes",
+    "servings": "1 serving",
+    "difficulty": "Easy",
+    "ingredients": [
+      {
+        "category": "Bumbu (blend of spices)",
+        "items": [
+          "0.75 clove garlic, whole",
+          "0.5 medium shallot, quartered",
+          "0.5 red hot chili pepper, or less for less spicy",
+          "1¼ tsp fresh ginger, peeled",
+          "¾ tsp vegetable oil",
+          "¼ + ⅛ tsp fresh turmeric",
+          "¼ + ⅛ tsp ground coriander"
+        ]
+      },
+      {
+        "category": "Soup base",
+        "items": [
+          "¾ tsp vegetable oil",
+          "0.5 medium potato, peeled, cut into ½cm slices",
+          "1 cup water",
+          "0.25 vegetable bouillon cube",
+          "⅓ cup + 1½ Tbsp canned full-fat coconut milk",
+          "1.3 oz dry ramen noodles"
+        ]
+      },
+      {
+        "category": "Vegan 'chicken'",
+        "items": [
+          "¾ tsp vegetable oil",
+          "1.8 oz mock 'chicken' pieces",
+          "0.75 clove garlic, thinly sliced",
+          "0.5 medium shallot, minced",
+          "½ tsp chicken spice mix"
+        ]
+      },
+      {
+        "category": "Garnish",
+        "items": [
+          "2½ Tbsp fresh bean sprouts",
+          "1 Tbsp fresh cilantro (optional)",
+          "0.25 stalk green onion, sliced",
+          "1½ tsp crispy fried onions (optional)"
+        ]
+      }
+    ],
```

Comment:
I think there's an inconsistency in how the ingredients are stored. An array with all the ingredients listed would suffice.

Reply:
```
"ingredients": [
      "½ cup + 1 Tbsp (135 g) canned diced tomatoes",
      "0.67 red bell pepper, chopped",
      "0.33 medium red onion, finely chopped",
      "0.67 red hot chili pepper, seeds removed, minced",
      "1 clove garlic, minced",
      "2 tsp (11 g) tomato paste",]
      
```

Like this would do